magefiles/installation/oauth.go

```diff
@@ -0,0 +1,120 @@
+package installation
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	retry "github.com/avast/retry-go/v4"
+	"github.com/devfile/library/pkg/util"
+	ocpOauth "github.com/openshift/api/config/v1"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+const (
+	HTPASSWD_FILE_NAME      = "ci.htpaswd"
+	DEFAULT_IDP_SECRET_NAME = "htpass-secret"
+	DEFAULT_IDP_NAME        = "ci_identity"
+	DEFAULT_OAUTH_NAME      = "cluster"
+	DEFAULT_HTPASSWD_BIN    = "htpasswd"
+)
+
+var (
+	randomOCPUserPass = util.GenerateRandomString(10)
+	randomOCPUserName = util.GenerateRandomString(6)
+	httpaswdArgs      = []string{"-c", "-B", "-b", HTPASSWD_FILE_NAME, randomOCPUserName, randomOCPUserPass}
```

Comment:
nitpic typo, but we should look for all places that have `httpaswd` and change `htpasswd`

Reply:
@Dannyb48 Not need anymore: I replace htpasswd with: https://github.com/redhat-appstudio/e2e-tests/pull/284/files#diff-1dfcb4f8bebb12bcfa1e480cf13a37f9bf1198cd29b988b73d3fb3398c1092b8R38